src/Modules/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/languageSettings/languageVerifier/languageVerifierGrid/index.jsx

```diff
@@ -14,11 +14,9 @@ class LanguageVerifierGridPanel extends Component {
     }
 
     uncollapse(id) {
-        setTimeout(() => {
-            this.setState({
-                openId: id
-            });
-        }, this.timeout);

```

Comment:
If you remove `setTimeout` I'd assume that `this.timeout` would also be unnecessary. Where does it come from? I only ever see it being read.